WinUIGallery/ControlPages/ProgressRingPage.xaml.cs

```diff
@@ -43,5 +44,45 @@ private void ProgressValue_ValueChanged(Microsoft.UI.Xaml.Controls.NumberBox sen
             }
         }
 
+        private void Background_SelectionChanged1(object sender, SelectionChangedEventArgs e)
+        {
+            string colorName = e.AddedItems[0].ToString();
+            bool showBackgroundProperty = false;
+            switch (colorName)
+            {
+                case "Transparent":
+                    ProgressRing1.Background = new SolidColorBrush(Colors.Transparent);
+                    break;
+                case "LightGray":
+                    ProgressRing1.Background = new SolidColorBrush(Colors.LightGray);
+                    showBackgroundProperty = true;
+                    break;
+
+                default:
+                    throw new Exception($"Invalid argument: {colorName}");
+            }
+            RevealBackgroundProperty1.IsEnabled = showBackgroundProperty;
+        }
+
+        private void Background_SelectionChanged2(object sender, SelectionChangedEventArgs e)
```

Comment:
Could we refactor this to determine only have one handler that selects the correct progressring based on the sender?

Reply:
I'm not sure how we can get that information from the sender. Any idea?

Reply:
but I agree, let's figure this out before merging it

Reply:
We can either set the tag property on the ComboBox and retrieve that through the sender property or doing something like this:

```c#
private void Background_SelectionChanged2(object sender, SelectionChangedEventArgs e)
    var progressRing = sender == BackgroundComboBox1 ? ProgressRing1 : ProgressRing2;
    // Color progress ring accordingly
}
```

Reply:
That's perfect! Thank you @chingucoding. I just pushed the change.